CONTRIBUTING_A_NEW_QUEST.md

```diff
@@ -0,0 +1,413 @@
+
+## Designing the form
+
+As mentioned, the user interface must leave no space for misunderstandings, it must be concise and quick and easy to use. Also sounds obvious, but you will quickly find out that a balance must be found between covering all the edge cases and designing the form to be as straightforward and clutterless as possible.
+
+- Design the main form clutter-free so that it is straightforward for the majority of use cases.
+- Allow to answer popular edge cases, but don't clutter up the main form with that. A good pattern is to move such answers into the "Other answers..." menu. I.e. look at the opening hours quest.
+- Don't rely on the "leave a note" fallback too much. It is not intended and does not work as a regular answer but is designed to cover the case that the question was invalid itself because it was based on wrong data like i.e. the place does not exist anymore.
+- The information the user should fill in should be as atomic as possible. Users are impatient, i.e. do not let them fill out a whole address with street name etc. when just the house number is fine too.
+- "A picture is worth a thousand words": Often the term for certain things may not be enough to convey the meaning of certain predefined answers. Do you know what a wheelbender is? You will know if you see the photo. 
+
+Considerations about the edge cases to consider, how the design could look like and finding good representative photos or icons that match in style is also part of the preparational work that can be done without programming knowledge.
+
+## New photos
+
+Some quests will require photos for their interface.
+
+Photos must be available on open license - so not every photo found on the Internet is usable. You can either take your own photo or find an existing freely licensed photo.
+
+Good places to find freely licensed images are [Geograph](https://www.geograph.org.uk/) and [Wikimedia Commons](https://commons.wikimedia.org/).
+
+You can also take your own photos, a standard smartphone camera is good enough. And sometimes a highly specific image is needed.
+
+In StreetComplete many images have unusual composition. Often it is necessary to leave space for a label at the bottom.
+
+Images should be free of visual debris, not misleading. Though it is fine to use an image not strictly matching what is depicted, as long as it is clear. For example, [a permanent pile of soil blocking road](https://wiki.openstreetmap.org/wiki/Tag:barrier%3Ddebris) is illustrated by a [temporary landslide](https://commons.wikimedia.org/wiki/File:Landslide_on_OR_42S_(46849629014).jpg). This is OK as images are illustrative.
+
+Photos go to a different folder than SVGs: they can be used directly by the build process so put them into folders
+  - [mdpi](app/src/main/res/drawable-mdpi) (typically 128 x 128 pixels, for rectangular ones smaller dimension should be 128 pixels)
```

Comment:
This is more complex, see https://github.com/streetcomplete/StreetComplete/issues/3725